logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -36,7 +37,8 @@ class ProtoContentMapperImpl(
     private val availabilityMapper: AvailabilityStatusMapper = MapperProvider.availabilityStatusMapper(),
     private val encryptionAlgorithmMapper: EncryptionAlgorithmMapper = MapperProvider.encryptionAlgorithmMapper(),
     private val idMapper: IdMapper = MapperProvider.idMapper(),
-    private val messageMentionMapper: MessageMentionMapper = MapperProvider.messageMentionMapper(),
+    private val userId: UserId? = null,
```

Comment:
```suggestion
    private val selfUserId: UserId? = null,
```

Reply:
Why is the `userId` optional? We will crash if it's not set.

Reply:
Because we only need it for mentions and we are sure that it's not null. 
It won't crash for other cases since it's not used.

Reply:
done